appveyor.yml

```diff
@@ -3,6 +3,7 @@
 
 init:
   - git config --global core.autocrlf input
+  - cmd: set NODE_OPTIONS=--max-old-space-size=3072
```

Comment:
It seems that ts-node that the new balena-config-karma uses needs more memory to compile.
Perhaps smaller values could also work, but I didn't see a benefit on experimenting and just used what I found in a appveyor help thread.